modules/microshift-install-rhel-image-mode-build-image.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-build-image_{context}"]
+= Building the bootc image
+
+Build your {op-system-base-full} that contains {microshift-short} as a bootable container image by using a Containerfile.
+
+[NOTE]
+====
+Image mode for {op-system} is technology preview. Using this bootable container image in production environments is not supported.
+====
+
+.Prerequisites
+* A {op-system-base-full} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images and running containers.
+* You are logged into the {op-system} 9.4 host using the user credentials that have `sudo` permissions.
+* The `rhocp` and `fast-datapath` repositories are accessible in the host subscription, but do not necessarily need to be enabled on the host.
+* You have a remote registry such as link:https://quay.io[Red Hat quay] for storing and accessing bootc images.
+
+.Procedure
+
+. Create a Containerfile and include the following instructions:
```

Comment:
include following instructions? 
```suggestion
. Create a Containerfile including the following commands:
```

Reply:
Yes, per the RHEL doc team review. It is consistent with how they talk about this, so if the user goes there for additional information, it will be the same. WDYT?

Reply:
It sounds strange to me, but OK, if they insist

Reply:
It looks like it's in the Podman docs as well, so I that seems to be the RH way. Okie dokie!